css/templates/e-commerce/page-vars.css

```diff
@@ -15,12 +15,27 @@
  */
 
 :root {
+  --h1: 3.5rem;
```

Comment:
take a look at the typography classes (also displayed on components page) and see if it makes sense to use/override them